app/assets/javascripts/chf_ga_events.js

```diff
@@ -0,0 +1,14 @@
+// Sufia/hyrax JS for sending event tracking to Google Analytics is a bit inflexible.
+// https://github.com/samvera/hyrax/blob/1e504c200fd9c39120f514ac33cd42cd843de9fa/app/assets/javascripts/hyrax/ga_events.js
```

Comment:
I think if we're not using the upstream behavior it'd be better to override so we can't accidentally use the id and start receiving weird / bad analytics data. Do you think there is a benefit to keeping the upstream code in play?